server/src/db/index.ts

```diff
@@ -1,6 +1,10 @@
-import { Post } from "@app/models/domain";
-import posts from "./content";
+import { Post } from '@app/models/domain';
+import posts from './content';
 
-export const get = (id:string): Post => {
-  return posts[id]
-}
\ No newline at end of file
+export const get = (id: string): Post => {
+  return posts[id];
+};
+
+export const getAll = (): string[] => {
```

Comment:
Given the title, I believe that this should return all posts (not only their IDs)